src/library/columncache.cpp

```diff
@@ -154,3 +269,25 @@ void ColumnCache::slotSetKeySortOrder(double notationValue) {
     // Replace the existing sort order
     m_columnSortByIndex[keyColumnIndex] = keySortSQL;
 }
+
+const QString& ColumnCache::columnName(Column column) const {
+    DEBUG_ASSERT(static_cast<std::size_t>(column) < std::size(kColumnPropertiesByEnum));
+    DEBUG_ASSERT(kColumnPropertiesByEnum[column].pName);
+    return *kColumnPropertiesByEnum[column].pName;
+}
+
+QString ColumnCache::columnTitle(Column column) const {
+    DEBUG_ASSERT(static_cast<std::size_t>(column) < std::size(kColumnPropertiesByEnum));
+    return QCoreApplication::translate(
+            "PredefinedColorPalettes", kColumnPropertiesByEnum[column].pTrTitle);
```

Comment:
this should be `"ColumnProperties"`, right?

Reply:
Oh yes that was a regression. Thank you for the finding.